### PR TITLE
Another layout for badges

### PR DESCRIPTION
**Pull request by maxmeyer**
## Summary

Another layout for the build badges in the readme

## Details

As pointed out in https://github.com/cucumber/aruba/pull/469 I think having a talble for the build status of "master" and "still" is easier to read. But I see the point in having some kind of a "sign post" for outsiders in what kind of state the project is.

This is why I moved up the "build" bages of the "master" branch.

I also added more details to clarify the use of the branches.

## Screenshots (if appropriate):

https://github.com/cucumber/aruba/tree/feature/other-layout-for-badges

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (cleanup of codebase withouth changing any existing functionality)
- [x] Update documentation

## Checklist:

- [ ] I've added tests for my code
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by junaruga** [comment]
Maybe you are a maintainer and you are going to fix current build failure/error by yourself.
I do not disagree with this.

**Comment by maxmeyer** [comment]
> Maybe you are a maintainer and you are going to fix current build failure/error by yourself.

What do you mean with this?

File: README.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-[![still Wiindows Build Status](https://ci.appveyor.com/api/projects/status/jfo2tkqhnrqqcivl/branch/still?svg=true&passingText=still%20windows%20passing&failingText=still%20windows%20failing&pendingText=master%20windows%20pending)](https://ci.appveyor.com/project/cucumberbdd/aruba/branch/still)
-([still](https://github.com/cucumber/aruba/tree/still))
-
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/cucumber/aruba/master/LICENSE)
 [![Docs](https://img.shields.io/badge/cucumber.pro-aruba-3d10af.svg)](https://app.cucumber.pro/projects/aruba)
 [![Gem Version](https://badge.fury.io/rb/aruba.svg)](http://badge.fury.io/rb/aruba)
 [![Dependency Status](https://gemnasium.com/cucumber/aruba.svg)](https://gemnasium.com/cucumber/aruba)
 [![Code Climate](https://codeclimate.com/github/cucumber/aruba.svg)](https://codeclimate.com/github/cucumber/aruba)
 [![Support](https://img.shields.io/badge/cucumber-support-orange.svg)](https://cucumber.io/support)
+<br>
```
Comment (junaruga) [inline README.md]: `<br>` HTML tag? I do not know that it makes sense for Markdown format.

**Comment by junaruga** [comment]
I think as you are a maintainer, it is good to update `README.md` and the build status as you like. Because you are a person that often see the build status.
Your opinion should be prioritized.
So, I will accept your design.

File: README.md (hunk 1 above)
Comment (maxmeyer) [inline README.md]: Good catch. I was not sure what the parser will do with a newline. So I took the "happy path" and just added <br> to make sure the build flags are on "their" own.

**Comment by maxmeyer** [comment]
Ah ok. I was unsure if I said something impolite which I oversaw. Thanks for the feedback. 

But anyway I see your point and I was not that comfortable with having the build status at the very end of the readme. Given this version in this PR it seems to be a good compromise for aruba developers and aruba users not being "disturbed" by information not interesting for that party.

**Pull request merged**